test/e2e/info_test.go

```diff
@@ -38,21 +37,9 @@ var _ = Describe("Podman Info", func() {
 		Expect(session.ExitCode()).To(Equal(0))
 
 	})
-	It("podman system info json output", func() {
-		session := podmanTest.Podman([]string{"system", "info", "--format=json"})
-		session.WaitWithDefaultTimeout()
-		Expect(session.ExitCode()).To(Equal(0))
-
-	})
-	It("podman info --format JSON GO template", func() {
-		session := podmanTest.Podman([]string{"info", "--format", "{{ json .}}"})
-		session.WaitWithDefaultTimeout()
-		Expect(session.ExitCode()).To(Equal(0))
-		Expect(session.IsJSONOutputValid()).To(BeTrue())
-	})
 
 	It("podman info --format GO template", func() {
-		session := podmanTest.Podman([]string{"info", "--format", "{{ .Store.GraphRoot }}"})
+		session := podmanTest.Podman([]string{"info", "--format", "{{.Store.GraphRoot}}"})
```

Comment:
Please pick a field that you can verify the value as well as exit code == 0.  Thanks

Reply:
fixed